### ISSUE TITLE
tldr;

### ISSUE THREAD

**Issue by iteles** [dwyl/contributing#28]
These are the things I feel like _shouting_ at people. The things that get done for the first couple of weeks and then fall by the wayside and **that can't happen**. The things I remind people to do _over and over_ again but seem to only be done to varying degrees and by a handful of people:
- Create an issue _before_ you start working on anything
  - Discuss your proposed implementation in that issue
  - **Add a time estimate** to it
- Once there is _at least one comment agreeing the feature is needed_, start working on it _by adding to the readme *_first***
- Link all commits to a relevant issue
- Each PR should:
  - Solve _only one_ problem: fix one bug, implement one small feature or part thereof
  - Have a clear and simple description which _also_ links to the issue it solves
  - Add to the documentation or explain why it shouldn't
  - Include full tests for any code that is introduced
  - Be assigned to the designated QA when ready for review

Adds to #12

**Comment by nelsonic** [comment]
agreed. please PR this into the main Readme so other people can be aware of it. 👍

**Comment by jay-meister** [comment]
@nelsonic for me is the most important thing but it is missing from the readme 😕

**Comment by nelsonic** [comment]
@JMurphyWeb agree 100%. I look _forward_ to this being in the Readme. 
it could even be the "Table of Contents" which then links to the more expanded sections.
Happy for ***anyone*** `else` to help improve this doc so it's clearererer to everyone. 👍

**Comment by jay-meister** [comment]
@nelsonic I like your suggestion.. (is that hint that you want me to PR this into the readme?)

I am happy to do so, but my point is more that there is no TLDR, not what the content of the TLDR should be. Are the bullet points above discussed & verified?

**Comment by nelsonic** [comment]
@JMurphyWeb if they were written by @iteles they are verified by `default` 😉 
but joking a side, yes, the bullet points above are a great starting point. 👍 
If we could get them PR'd into the Readme soon it would be _superb_! 🌈
